src/plugins/intel_npu/src/plugin/npuw/weights_bank.cpp

```diff
@@ -43,24 +43,63 @@ ov::Tensor Bank::get(const ov::Tensor& tensor) {
     return tensor;
 }
 
-std::shared_ptr<Bank> BankManager::getBank(const std::string& bank_name) {
+ov::Tensor Bank::get(const ov::Tensor& tensor, const std::string& device) {
+    if (!tensor) {
+        OPENVINO_THROW("Uninitialized tensor in weights bank allocation!");
+    }
+
+    if (device != "CPU" && device != "NPU") {
+        OPENVINO_THROW("Unsupported device in weights bank allocation: ", device);
+    }
```

Comment:
OK for now but probably, in the future, we should just use the "CPU" path for such cases